abe/resource_models/subscription_resources.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
```

Comment:
The shebang is unnecessary unless the module doubles as a command-line script, which I think isn't true here. (I know that lots of the rest of the codebase follows this pattern. In this case I vote for incremental improvement over consistency.)

Reply:
OK, I usually like including it to mark whether a file is written for Python 2 or Python 3, since I often work with both. In this case, it just slipped in when this file was copied from the label resources document.

Reply:
Ah, that's a point. You could also include `# -*- coding: utf-8 -*-` on only Python 2 files, since only those need that, but's kind of obtuse.